Resources/Textures/_DV/Interface/Radial/RCD/attributions.yml

```diff
@@ -0,0 +1,11 @@
+# Delta-V RCD sprites for the interaction wheel
+
+- files: ["airlock.png","airlocks.png","directional.png","directional_reinforced.png","glass_airlock.png","window.png","window_reinforced.png"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Created by SubspaceEmber for Delta-V"
```

Comment:
Missing proper attribution for original sprites.
At the very least airlock.png appears to be based on the CC-BY-SA-3.0 sprite for closed default airlocks